install.sh

```diff
@@ -120,6 +120,42 @@ nvm_detect_profile() {
   fi
 }
 
+#
+# Check whether the user has any globally-installed npm modules in their system
+# Node, and warn them if so.
+#
+nvm_check_global_modules() {
+  local NPM_GLOBAL_MODULES
+  NPM_GLOBAL_MODULES=$(npm list -g --depth=0 | sed '/ npm@/d')
+  
+  local MODULE_COUNT
+  MODULE_COUNT=$(
```

Comment:
Ah, didn't know I should make install friendly to pre-existant `nvm`s! I'll do so.

As for no-npm, that's just an oversight. Should be trivial, will add.